models/dbschema.js

```diff
@@ -19,6 +19,7 @@ var BuildSchema = new Schema ({
   gitBranch: String,
   gitSHA: String,
   dockerDigest: String,
+  commits: String,
```

Comment:
I have taken out the commits attributes for now,  But I think we will need a easy way to track it . Show the delta commits in the build is a very useful feature IMHO